nodes/hardfork-process.md

```diff
@@ -31,34 +31,14 @@ The general process can be broken down into several components:
 - Testing of the features (happens on testnets first prior to activating on
   mainnet in order to ensure the network can upgrade securely).
 
-The two testnets were hardforks are deployed on are:
+The two testnets where hardforks are deployed are:
 
 - [Arabica devnet](./arabica-devnet.md)
 - [Mocha testnet](./mocha-testnet.md)
 
-### Mocha hardfork
+### Lemongrass hardfork
 
-Celestia is planning the Mocha Hardfork upgrade on the Mamaki Testnet.
-This hardfork is unique as it will reset the Mamaki network to block 0
-while maintaining the existing state and also will rename Mamaki to Mocha.
+The Lemongrass hardfork is the first consensus layer breaking change since Celestia's mainnet genesis block. The Lemongrass hardfork includes all of the CIPs listed in [CIP-17](https://github.com/celestiaorg/CIPs/blob/main/cips/cip-17.md). The Lemongrass hardfork will be executed on Arabica, then Mocha, then Mainnet Beta. The hardfork will take place at an "upgrade height" that will be coordinated offline on a per-network basis. The upgrade heights will be announced in advance (follow Telegram and Discord) to give node operators time to download and start a compatible binary prior to the upgrade height.
```

Comment:
```suggestion
The Lemongrass hardfork is the first consensus layer breaking change since Celestia's Mainnet Beta genesis block. The Lemongrass hardfork includes all of the CIPs listed in [CIP-17](https://github.com/celestiaorg/CIPs/blob/main/cips/cip-17.md). The Lemongrass hardfork will be executed on Arabica, then Mocha, then Mainnet Beta. The hardfork will take place at an "upgrade height" that will be coordinated offline on a per-network basis. The upgrade heights will be announced in advance (follow Telegram and Discord) to give node operators time to download and start a compatible binary prior to the upgrade height.
```